src/reconcile/rec_col.c

```diff
@@ -8,8 +8,6 @@
 
 #include "wt_internal.h"
 
-// 9417 IGNORE asserts in this file. Column store is not related to BF-25011
```

Comment:
I'm following up with @ajmorton as to whether these // 9417 ignore comments are meaningful.

Reply:
No we can drop them. 
These were notes to myself when doing WT-9417 that should have been removed and slipped through review. There's a second one in `rec_child.c` that can also go